src/app/pages/interest-form/interest-form.scss

```diff
@@ -0,0 +1 @@
+@import 'forms';
```

Comment:
Can you move the `forms.scss` file into `/components/forms.scss` and import at the bottom of `main.scss` (like buttons, headers, etc)?  This file can then be deleted if no unique styling is necessary for this page.